code/obj/decal.dm

```diff
@@ -669,6 +669,21 @@ obj/decal/fakeobjects/teleport_pad
 	name = "QM Navigation Guide"
 	desc = "The quartermaster is in this direction."
 	icon_state = "guide_qm"
+	
+/obj/decal/tile_edge/floorguide/hop
+	name = "Head Of Personnel Navigation Guide"
+	desc = "The head of personnel's office is in this direction."
```

Comment:
```suggestion
	desc = "The Head of Personnel's office is in this direction."
```

Reply:
Oh you're right, the descriptions are pointless since you can't see them ingame. I think it was something old i had planned but scrapped.

Reply:
The floor decals code could in theory be extended to append the descriptions to the floor tile's `desc`. 🤔